webapp/locales/en.json

```diff
@@ -482,7 +482,7 @@
     "reports": {
       "empty": "Congratulations, nothing to moderate.",
       "name": "Reports",
-      "typeRowHeadline": "Type",
+      "typeRowHeadline": "Current status",
```

Comment:
I'd have to check, but I guess this is not the `Current status`... by `Type`, I think @Tirokk was referring to the `type` that used to be returned from the `backend` as in the `type of Resource that was reported`... in the `NotificationsTable`, I didn't have a header for this column...
I'll see what @alina-beck thinks about the `layout` though, since I trust her judgement more than any of ours :stuck_out_tongue_closed_eyes: